src/docfx/build/Build.cs

```diff
@@ -86,6 +86,11 @@ private static void Run(Context context)
                     context.ErrorBuilder,
                     context.PublishUrlMap.GetAllFiles(),
```

Comment:
>context.PublishUrlMap.GetAllFiles() [](start = 20, length = 35)

How about concat the additional resource files on the original scope?

Reply:
Actually it can't, for the additional resources are ONLY found after the static scope get built.

Reply:
right